mdn/kumascript.py

```diff
@@ -722,10 +724,33 @@ class KumaVisitor(BaseKumaVisitor):
     _default_attribute_actions = {None: 'ban'}
 
     def visit_a_open(self, node, children):
-        """Ensure that <a> open tags have an href element."""
+        """Validate and cleanup <a> open tags."""
         actions = self._default_attribute_actions.copy()
         actions['href'] = 'must'
-        return self._visit_open(node, children, actions)
+        actions['title'] = 'drop'
+        actions['class'] = 'keep'
+        converted = self._visit_open(node, children, actions)
+
+        # Convert relative links to absolute links
+        attrs = converted.attributes.attrs
+        if 'href' in attrs:
+            href = attrs['href'].value
+            if href and href[0] == '/':
+                attrs['href'].value = MDN_DOMAIN + href
```

Comment:
Just because I wanted to think about the single worst possible line of python ever:
`if getattr(attrs.get('href', {}), 'value', ' ') == '/':`